### DISCUSSION TITLE
Failed to connect to the host via ssh:

### DISCUSSION THREAD

**Discussion by DevOpsPa1** [learndevopsonline/learndevopsonline#1839] [Q&A]
![Screenshot (38)](https://github.com/learndevopsonline/learndevopsonline/assets/157614154/af64f360-72a5-4217-ade4-50985d0d6e9f)

**Comment by chaithanya1812** [comment]
Hi DevOpsPa1,

Start the workstation & share the workstation IP address
And also ensures that frontend server is up on running...

**Comment by DevOpsPa1** [comment]
Thank you for chaithanya support
